lib/tool_shed/webapp/frontend/src/components/pages/CitableRepositoryPage.vue

```diff
@@ -18,8 +16,17 @@ const error = ref<string | null>(null)
 
 async function update() {
     error.value = null
-    const { data } = await indexFetcher({ owner: props.username, name: props.repositoryName })
-    if (data instanceof Array) {
+    const { data, error: requestError } = await client.GET("/api/repositories", {
+        params: {
+            query: {
+                owner: props.username,
+                name: props.repositoryName,
+            },
+        },
+    })
+    if (requestError) {
+        error.value = requestError.err_msg
```

Comment:
Not sure it's always a JSON response with err_msg

Reply:
If we trust the schema it should :sweat_smile: 
But now that I remember I added the middleware probably none of this works as expected because all requests in the Tool Shed need to be wrapped in a try/catch, I think :thinking:

Reply:
The schema is for the app, but the proxy can answer with gateway timeouts, rate limits etc